bittide-shake/src/Clash/Shake/Vivado.hs

```diff
@@ -68,6 +69,8 @@ meetsTiming path = andM
   , fmap not $ inFile "Timing constraints are not met" path
   ]
 
+type GlobPattern = String
```

Comment:
Perhaps:

```haskell
type TclGlobPattern = String
```

And make sure to point out https://www.tcl.tk/man/tcl8.6/TclCmd/glob.htm